serverlessworkflow/antora.yml

```diff
@@ -77,3 +78,9 @@ asciidoc:
     openshift_application_data_services_service_account_url: https://console.redhat.com/application-services/service-accounts
     openshift_application_data_services_service_registry_url: https://console.redhat.com/application-services/service-registry
     openshift_application_data_services_apache_kafka_url: https://console.redhat.com/application-services/streams/kafkas
+    camel_url: https://camel.apache.org/
+    # must align this version
+    camel_extensions_url: https://camel.apache.org/camel-quarkus/2.14.x/reference/extensions
+    kaoto_url: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto
+
```

Comment:
These variables have been introduced in the Camel guide, which is 1.33 related. I don't think it will harm anything thou